pom.xml

```diff
@@ -44,6 +44,7 @@
         <che.docs.version>5.4.0-SNAPSHOT</che.docs.version>
         <che.lib.version>5.4.0-SNAPSHOT</che.lib.version>
         <che.version>5.4.0-SNAPSHOT</che.version>
+        <lsp4j.version>0.2.0-SNAPSHOT</lsp4j.version>
```

Comment:
It's not a problem for development phase. But it would be nice to do that before merging to master. Is that ok for you?